task/04-arrays-tasks.js

```diff
@@ -146,7 +146,7 @@ function getStringsLength(arr) {
  *    [ 1, 'b', 'c'], 0, 'x'  => [ 'x', 1, 'b', 'c' ]
  */
 function insertItem(arr, item, index) {
-   throw new Error('Not implemented');
+   return arr.splice(index, 0, item);
```

Comment:
Never change arguments. It is a potential bug. Why? `splice` operator change `arr` and external code will work with mutated array.